components/radio/stories/radio.stories.js

```diff
@@ -33,7 +37,7 @@ export default {
 			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
-				category: "Component",
+				category: "Content",
```

Comment:
It might be nice to have a `description` here, clarifying that "Name" is referring to the `name` attribute on the input. What do you think?

Reply:
Good idea. I've added that control description.